snmp/datadog_checks/snmp/data/profiles/chatsworth_pdu.yaml

```diff
@@ -570,6 +570,12 @@ metrics:
         column:
           OID: 1.3.6.1.4.1.30932.1.10.1.3.110.1.2
           name: cpiPduBranchId
+      - column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.2.10.1.3
+          name: cpiPduName
+        table: cpiPduTable
+        index_transform: "1:8"
```

Comment:
Note: feel free to comment if you think of another way to handle this feature or another syntax/way for indicating `index_transform: "1:8"`

Reply:
At first I thought this was a "mapping", i.e. "position 1 goes to position 8", but this seems to be more of a slice, i.e. "take indexes from position 1 to position 8" (where position starts at 0, I assume).

It's very low-level, and would need extra care in documentation (otherwise I don't expect anyone to use this, or to use this correctly? Eg the fact that `end` is excluded like in Python slices might trip up some people).

Maybe a more verbose API would be beneficial here…

```yaml
match_other_table_index:
  - start: 5
    end: 7
  - start: 2
    end: 3
```

Where `end` would be included (seems more natural and easy to account for in code).

Reply:
I though about the more verbose way. Can be a good alternative, and avoid having to do some parsing.